python/paddle/device/cuda/__init__.py

```diff
@@ -204,3 +205,69 @@ def stream_guard(stream):
             yield
         finally:
             stream = _set_current_stream(pre_stream)
+
+
+def get_device_properties(device=None):
+    '''
+    Return the properties of given CUDA device.
+
+    Args:
+        device(paddle.CUDAPlace() or int or str): The device, the ID of the device 
+            or the string name of device like 'gpu:x' which to get the properties of 
+            the device from. If device is None, the device is the current device. 
+            Default: None.
+
+    Returns:
+        _CudaDeviceProperties: the properties of the device which include ASCII string 
```

Comment:
`_CudaDeviceProperties` -> `_gpuDeviceProperties`? I guess you can fix this in the next PR with `test=document_fix`.。。